diff --git a/node_modules/@typescript-eslint/eslint-plugin/dist/rules/unbound-method.js b/node_modules/@typescript-eslint/eslint-plugin/dist/rules/unbound-method.js
index ad30633..4cd927f 100644
--- a/node_modules/@typescript-eslint/eslint-plugin/dist/rules/unbound-method.js
+++ b/node_modules/@typescript-eslint/eslint-plugin/dist/rules/unbound-method.js
@@ -171,7 +171,7 @@ exports.default = (0, util_1.createRule)({
                 checkIfMethodAndReport(node, services.getSymbolAtLocation(node));
             },
             ObjectPattern(node) {
-                if (isNodeInsideTypeDeclaration(node)) {
+                if (isException(node) || isNodeInsideTypeDeclaration(node)) {
                     return;
                 }
                 let initNode = null;
@@ -319,4 +319,13 @@ function isSafeUse(node) {
     }
     return false;
 }
+
+// CUSTOM
+function isException(node) {
+  return (
+    node.parent.init?.type === utils_1.AST_NODE_TYPES.CallExpression
+    && node.parent.init.callee.name === 'require'
+    && node.parent.init.arguments[0].value.startsWith('node:')
+  );
+}
 //# sourceMappingURL=unbound-method.js.map
\ No newline at end of file
